lib/python/ray/graph.py

```diff
@@ -0,0 +1,49 @@
+# Utilities to deal with computation graphs
+
+import ray.internal.graph_pb2
+import graphviz
+
+def graph_to_dot(computation_graph, out_filename, view=False):
```

Comment:
maybe `out_filename` should be optional so that the user can choose to view it but not save it?